src/worldmap_ctrl.ts

```diff
@@ -58,6 +58,11 @@ const panelDefaults = {
   ignoreEmptyGeohashValues: false,
   ignoreInvalidGeohashValues: false,
   stickyLabels: false,
+  enableOverlay: false,
+  overlayUrl: '',
+  overlayOpacity: 0.5,
+  overlayLatRange: '0,10',
+  overlayLngRange: '0,20',
```

Comment:
If you don't mind, you can well use long names here, like `overlayRangeLatitude` vs. `overlayRangeLongitude`.

Reply:
Done!